CommunityFridgeMapApi/tests/unit/test_fridge.py

```diff
@@ -15,6 +16,7 @@ def test_layer_test():
 
 
 def test_get_ddb_connection():
```

Comment:
Was there an error happening there without this?

Reply:
Yeah, so that test wasn't entirely self sufficient in that it required a default region to be specified on the machine that it is running on. If you remove your local aws config file from `~./aws/config` and run the tests you'll notice failures due to there not being a default region set.

Reply:
ooohhh, cool